CONTRIBUTING.md

```diff
@@ -173,7 +173,9 @@ We also build commonjs builds to run in node(for testing with jest or etc.) and
 Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+
```

Comment:
Not sure why these new lines are being added... 

I found a related issue from a while ago https://github.com/remarkjs/remark/issues/221

If it's a problem, we could raise it with the remark team